features/guidance/anticipate-lanes.feature

```diff
@@ -343,6 +343,38 @@ Feature: Turn Lane Guidance
             | waypoints | route    | turns                           | lanes                                  |
             | a,h       | ab,gh,gh | depart,roundabout-exit-5,arrive | ,slight right:false slight right:true, |
 
+    @anticipate @todo
+    Scenario: Anticipate with lanes in roundabout where we stay on the roundabout for multiple exits
+        # Scenario requires correct lane handling for clockwise roundabouts
+        Given the profile "lhs"
+        And the node map
+            |   |   | a |   |   |
+            |   |   | b |   |   |
+            | h | c |   | g |   |
+            |   |   |   |   |   |
+            |   | d |   | f |   |
+            |   |   | e |   |   |
+            | x |   |   |   | y |
+
+        And the ways
+            | nodes | turn:lanes:forward         | highway | junction   |
+            | ab    | slight_left\|slight_left   | primary |            |
+            | bg    |                            | primary | roundabout |
+            | gf    |                            | primary | roundabout |
+            | fe    |                            | primary | roundabout |
+            | ed    |                            | primary | roundabout |
+            | dc    | slight_left                | primary | roundabout |
+            | cb    |                            | primary | roundabout |
+            | ch    |                            | primary |            |
+            | ex    |                            | primary |            |
+            | dx    |                            | primary |            |
+            | gy    |                            | primary |            |
+            | fy    |                            | primary |            |
+
+        When I route I should get
+            | waypoints | route      | turns                           | lanes                                  |
+            | a,h       | ab,ch,ch   | depart,roundabout-exit-5,arrive | ,slight right:false slight right:true, |
```

Comment:
yes, here must be `slight left`, but the test fails because query returns an empty result, so i set `todo` state.

I will try to find out the reason during the next week. @MoKob showed me the starting place https://github.com/Project-OSRM/osrm-backend/blob/master/src/extractor/guidance/turn_lane_matcher.cpp#L54-L103